src/api/common_api.js

```diff
@@ -1188,5 +1188,10 @@ module.exports = {
                 }
             },
         },
+
+        storage_class_enum: {
+            type: 'string',
+            enum: ['STANDARD', 'FS_ARCHIVE', 'S3_GLACIER']
```

Comment:
I think its good, going with enum means we can add other stuff later and yes we could expose this on the BackingStores at some point. I think marking as class instead of specific storage types lets you control better how you want to use class X (and maybe for me class X would be used differently).

And again, we can always add more values to the enum if there is a need for more class seperation

Reply:
@guymguym If I understand your question is whether we want a big list of all supported storage classes of all different vendors or only the storage classes levels applying to all possible. I have to admit that using levels can make it more clear. I'm seeing that Google support 4 levels: STANDARD, NEARLINE, COLDLINE and ARCHIVE. maybe we should do the same?
https://cloud.google.com/storage/docs/storage-classes

Reply:
Yeah @jackyalbo that was my though too. I wonder if trying to define "abstract" storage classes will be successful, or we will find ourselves struggling to decide how to map the different providers classes to our abstract classes. For example, what should "S3 Glacier Instant Retrieval" map to vs "S3 Glacier Flexible Retrieval" vs. "S3 Glacier Deep Archive"...

Reply:
I think maybe we can have more generic names here and then later on can introduce more specific names if a need arises.

The only issue I think that can occur with the usage of generic names like `ARCHIVE` is that if we have a backing store supporting 2 types of archiving then we will need a way to disambiguate.

How about having `STANDARD` and `ARCHIVE` for now? Here `STANDARD` and `undefined` (i.e unset) storage class can be treated similarly.

Reply:
@utkarsh-pro I think your proposal is good. Lets go with STANDARD and ARCHIVE enums only for now. We can always add more later.

Reply:
We talked and agreed to align with the storage class enum of S3, and for now we will define just STANDARD and GLACIER and add more as needed.